redis/src/main/scala/zio/redis/api/Streams.scala

```diff
@@ -759,7 +759,7 @@ trait Streams {
   }
 }
 
-private object Streams {
+private[redis] object Streams {
```

Comment:
Why is this necessary?

Reply:
Unnecessary. Just for consistency.
revert it